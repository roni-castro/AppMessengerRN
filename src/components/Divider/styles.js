import {StyleSheet} from 'react-native';

export const styles = StyleSheet.create({
  divider: {
    borderBottomWidth: 1,
    backgroundColor: '#c3c4c4',
    marginVertical: 10,
  },
});
