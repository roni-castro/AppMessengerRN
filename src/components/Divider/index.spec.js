import React from 'react';
import {render} from '@testing-library/react-native';
import Divider from '.';

describe('Unit | Divider', () => {
  it('renders the component correctly and with the default style', () => {
    const wrapper = render(<Divider />);
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
