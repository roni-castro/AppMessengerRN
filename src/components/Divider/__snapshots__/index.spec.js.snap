// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unit | Divider renders the component correctly and with the default style 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#c3c4c4",
      "borderBottomWidth": 1,
      "marginVertical": 10,
    }
  }
/>
`;
